src/components/Article/SingleArticle.jsx

```diff
@@ -44,7 +45,10 @@ const SingleArticle = () => {
           <main>
             {article.body}
           </main>
-          <p className="footer right smaller gray">{article.votes} Votes</p>
+          <footer className="footer">
+            <Vote article_id={article_id} setAddVote={setAddVote} />
+            <p className="right smaller gray">{article.votes + addVote} Votes</p>
+          </footer>
         </article>
         <CommentList article_id={article_id} />
       </body>
```

Comment:
Currently showing an error in the console because of this body tag
As the error says, you can't have the body tag inside a div and a react project is in one bit div with the class of "root". You can check this by inspecting the elements of your app
Can you fix this error please